src/components/Message/renderText/renderText.tsx

```diff
@@ -35,6 +35,13 @@ export const defaultAllowedTagNames: Array<keyof JSX.IntrinsicElements | 'emoji'
   // custom types (tagNames)
   'emoji',
   'mention',
+  'table',
+  'thead',
+  'tbody',
+  'th',
+  'tr',
+  'td',
+  'tfoot',
```

Comment:
Can we move these above the "// custom types" comment as these are valid HTML elements?

Reply:
Oh, sure - missed that